language/language_samples.rb

```diff
@@ -12,58 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def sentiment_from_text project_id:, text_content:
+def sentiment_from_text text_content:
   # [START sentiment_from_text]
-  # project_id   = "Your Google Cloud project ID"
   # text_content = "Text to run sentiment analysis on"
 
   require "google/cloud/language"
 
-  language  = Google::Cloud::Language.new project: project_id
-  document  = language.document text_content
-  sentiment = document.sentiment
+  language = Google::Cloud::Language.new
+  response = language.analyze_sentiment content: text_content, type: :PLAIN_TEXT
```

Comment:
I believe `type: :PLAIN_TEXT` will be updated as the default value for this operation. @remi has more context here. 

One key requirement is to not have this parameter be required.